Assets/MixedRealityToolkit.Extensions/QRCodesTracker/Scripts/QRCodesManager.cs

```diff
@@ -89,8 +87,17 @@ public bool TryGetLocationForQRCode(Guid id, out Matrix4x4 location)
             location = Matrix4x4.identity;
 
 #if WINDOWS_UWP
-            var coordinateSystem = SpatialGraphInteropPreview.CreateCoordinateSystemForNode(id);
-            return TryGetLocationForQRCode(coordinateSystem, out location);
+            try
+            {
+                var coordinateSystem = SpatialGraphInteropPreview.CreateCoordinateSystemForNode(id);
+                return TryGetLocationForQRCode(coordinateSystem, out location);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"Exception thrown creating coordinate system for qr code id: {e.ToString()}");
```

Comment:
>qr code id: [](start = 81, length = 11)

I think either update this string to be "exception: " or also log the qr code id as well.